tools/cockpit.spec

```diff
@@ -191,6 +223,11 @@ install -p -m 644 tools/cockpit.pam $RPM_BUILD_ROOT%{_sysconfdir}/pam.d/cockpit
 rm -f %{buildroot}/%{_libdir}/cockpit/*.so
 install -D -p -m 644 AUTHORS COPYING README.md %{buildroot}%{_docdir}/cockpit/
 
+%if %{cockpit_enable_python} && %{build_basic}
+%py3_install_wheel %{name}-%{version}-py3-none-any.whl
+mv -t %{buildroot}%{_libexecdir} %{buildroot}%{_bindir}/cockpit-askpass
```

Comment:
Both of this needs to happen in Makefile.am, this knowledge does not belong into the spec. It would have to be repeated in the Debian, arch, gentoo, etc. packaging, and breaks `make install` -- that currently doesn't install /usr/bin/cockpit-bridge or c-askpass at all.

I know that this will install the py bridge for !build_basic as well, but we already have to filter out files for the C bridge anyway.

Reply:
See above comment about this.